.github/make_workflows.rb

```
@@ -0,0 +1,151 @@
+      branches: [ main ]
+    pull_request:
+      branches: [ main ]
+  jobs:
+    build:
+      runs-on: XXX
+      steps:
+      - uses: actions/checkout@v4
+      - name: make check
+        run: make check
+  YAML
+  spec = YAML.load(template)
```

RuboCop (1.73)

[convention] Prefer using `YAML.safe_load` over `YAML.load`.

(Security/YAMLLoad)